docker/dbeaver/config/initial-data-sources.conf

```diff
@@ -0,0 +1,46 @@
+{
+  "folders": {},
+  "connections": {
+    "postgresql-template-1": {
+      "provider": "postgresql",
+      "driver": "postgres-jdbc",
+      "name": "PostgreSQL (Template)",
+      "save-password": false,
+      "show-system-objects": false,
+      "read-only": true,
+      "template": true,
+      "configuration": {
+        "host": "localhost",
+        "port": "5432",
+        "database": "postgres",
+        "url": "jdbc:postgresql://localhost:5432/postgres",
+        "type": "dev",
+        "provider-properties": {
+          "@dbeaver-show-non-default-db@": "false"
+        }
+      }
+    },
```

Comment:
Same here as with the h2 part in the other file: what is this postgres connection for? There's no postgres running on the server, unless dbeaver has its own internal DB for user management or something..?

Reply:
Seems just to be an example database. I removed it.